src/main/kotlin/no/nav/familie/ef/søknad/api/dto/søknadsdialog/Aktivitet.kt

```diff
@@ -1,27 +1,30 @@
 package no.nav.familie.ef.søknad.api.dto.søknadsdialog
 
 data class Aktivitet(
-        val arbeidsforhold: List<Arbeidsforhold>?,
+        val arbeidsforhold: List<Arbeidsgiver>?,
         val arbeidssøker: Arbeidssøker?=null,
         val firma: Firma? = null,
         val hvaErDinArbeidssituasjon: ListFelt<String>,
         val underUtdanning: UnderUtdanning? = null,
-        val etablererEgenVirksomhet: TekstFelt? = null
-
+        val etablererEgenVirksomhet: TekstFelt? = null,
+        val egetAS: List<Aksjeselskap>?
 )
 
+data class Aksjeselskap ( // TODO - ikke mappet
+        val navn: TekstFelt?= null,
+        val arbeidsmengde: TekstFelt?= null
+)
 
-
-data class Arbeidsforhold(
-        val arbeidsmengde: TekstFelt,
-        val fastStilling: TekstFelt,
-        val harSluttDato: BooleanFelt,
-        val sluttdato: DatoFelt,
-        val navn: TekstFelt,
-        val firma: Firma? = null,
-        val react_key: String?
+data class Arbeidsgiver(
+    val ansettelsesforhold: TekstFelt,
+    val arbeidsmengde: TekstFelt,
+    val id: String,
```

Comment:
Er dette id-er som skal brukes av backend eller brukes det bare av UI ? Både Arbeidsgiver og Aksjeselskap har dette i frontend for å holde de unike innenfor lista, men her er det bare Arbeidsgiver som bruker den